zookeeper/metadata.json

```
@@ -0,0 +1,7 @@
+{
+  "hub": {
+    "categories": [
+      "message-queues"
```

```suggestion
      "databases-and-storage"
```

(it takes a bit to unpack the tech lingo on many official zookeeper descriptions, but it's a distributed key-value store, _not_ a message queue :eyes:)